evap/evaluation/tools.py

```diff
@@ -94,7 +95,11 @@ def mix(a, b, alpha):
     return alpha * a + (1 - alpha) * b
 
 
-def get_answers(course, contribution, question):
+def get_all_textanswers(course, contribution, question):
+    return get_answers(course, contribution, question, exclusion=[])
+
+
+def get_answers(course, contribution, question, exclusion=['', 'N']):
```

Comment:
is there a difference between the textanswers of `get_all_textanswers` and answers from `get_answers`? if not, please remove/add the text to the other name.

also, i think it is not good to have a "get results" and a "get all results" method. it is totally not obvious that the "get results" excludes some of the results by default. i think it should be "get results" and "get filtered results".